src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/PdoSessionHandlerTest.php

```diff
@@ -332,6 +332,7 @@ public function testUrlDsn($url, $expectedDsn, $expectedUser = null, $expectedPa
     public function provideUrlDsnPairs()
     {
         yield ['mysql://localhost/test', 'mysql:host=localhost;dbname=test;'];
+        yield ['mysql://localhost/test?unix_socket=socket.sock&charset=utf8mb4', 'mysql:host=localhost;dbname=test;unix_socket=socket.sock;charset=utf8mb4;'];
```

Comment:
`unix_socket` cannot be used with `host` or `port`. See https://www.php.net/manual/en/ref.pdo-mysql.connection.php

```suggestion
        yield ['mysql://localhost/test?unix_socket=socket.sock&charset=utf8mb4', 'mysql:dbname=test;unix_socket=socket.sock;charset=utf8mb4;'];
```